frontend/apps/filemanager/filemanagerfilesearcher.lua

```diff
@@ -17,12 +18,22 @@ local _ = require("gettext")
 local N_ = _.ngettext
 local T = require("ffi/util").template
 
-local FileSearcher = WidgetContainer:extend{
+local FileSearcher = InputContainer:extend{
     case_sensitive = false,
     include_subfolders = true,
     include_metadata = false,
 }
 
+function FileSearcher:init()
+    self:registerKeyEvents()
+end
+
+function FileSearcher:registerKeyEvents()
+    if Device:hasKeyboard() then
+        self.key_events.ShowFileSearch = { { "Shift", "F" }, args = true }
```

Comment:
`args = ""`
Will there be the second option, last search string?

Reply:
I don't think so... do you need one? I mean, it's not like anyone is trying to find the same file over and over again